examples/connector/java/src/main/java/connector/ConnectorServiceImpl.java

```diff
@@ -2,13 +2,20 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import fivetran_sdk.*;
+import connector.logging.FivetranLoggerFactory;
+import connector.logging.MessageOrigin;
 import fivetran_sdk.Record;
+import fivetran_sdk.*;
 import io.grpc.stub.StreamObserver;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+
+public class ConnectorServiceImpl extends SourceConnectorGrpc.SourceConnectorImplBase {
 
-import java.util.*;
+    private static final Logger LOGGER = FivetranLoggerFactory.getFivetranLogger(ConnectorServiceImpl.class, MessageOrigin.SDK_CONNECTOR);
```

Comment:
No no no we don't want to release production code to the public. Please undo this change and remove all the logging classes you added.

Reply:
reverted this